apps/storybook/docs/d_applicatief/d_voorbeelden/accordion-popover-menu/accordion-popover-menu.stories.ts

```diff
@@ -0,0 +1,344 @@
+
+const topLevelAccordionCss = `
+    .vl-accordion {background-color: white border-radius: 4px;}
+    .vl-accordion__content {background-color: rgb(248,249,252); padding: 0 15px; border-radius: 4px}
+    .vl-accordion__button-container {padding: 15px 15px 0 15px;}
+    .vl-accordion__subtitle {padding: 0 15px 0 15px;}
+`;
+const subAccordionCss = `
+    .vl-accordion {background-color: white; border-radius: 4px;}
+    .js-vl-accordion--open {background-color: rgb(248,249,252);}
+    .vl-accordion__button-container {padding: 15px;}
+    .js-vl-accordion--open > .vl-accordion__button-container {padding: 15px 15px 0 15px}
+    .vl-accordion__subtitle {margin: 0;}
+`;
+
+export const AccordionPopoverMenu = () => html`
+    <style>
+        .laaginfo {
+            display: flex;
+        }
+        .laaginfo__image {
+            flex-basis: 15%;
+            margin-right: 20px;
+        }
+        .laaginfo__table {
+            display: flex;
+            flex-basis: 100%;
+            flex-direction: column;
+        }
+        .laaginfo__table--row {
+            display: flex;
+            justify-content: space-between;
+        }
+        .laaginfo__table--row div {
+            flex-basis: 50%;
+            padding-bottom: 10px;
+        }
+        .panel {
+            border: 1px solid darkgray;
+            border-radius: 4px;
+        }
+        .panel > div:not(:last-child) {
+            border-bottom: 1px solid darkgray;
+        }
+        .panel > div:last-child {
+            border-radius: 4px;
+        }
+    </style>
+    <div class="panel">
+        <div>
+            <vl-accordion data-vl-toggle-text="Stedelijk woongebied" data-vl-custom-css=${topLevelAccordionCss}>
+                <span class="laaginfo">
+                    <div class="laaginfo__image">
+                        <img is="vl-image" class="laaginfo__image" src="cat.jpeg" alt="Example image" />
+                    </div>
+                    <div class="laaginfo__table">
+                        <div class="laaginfo__table--row">
+                            <div>Laagkenmerk</div>
+                            <div>Grondvlak</div>
+                        </div>
+                        <div class="laaginfo__table--row">
+                            <div>Categorie gebiedsaanduiding</div>
+                            <div>Wonen</div>
+                        </div>
+                        <div class="laaginfo__table--row">
+                            <div>Legendecode</div>
+                            <div>01109_XX</div>
+                        </div>
+                    </div>
+                </span>
+                <vl-annotation slot="subtitle">Lorem ipsum</vl-annotation>
+                <span slot="menu">
+                    <a is="vl-link" id="btn-acties1">
+                        <span is="vl-icon" data-vl-icon="nav-show-more-vertical"></span>
+                    </a>
+                    <vl-popover for="btn-acties1" placement="bottom-end">
```

Comment:
mss te overwegen om `distance="5"` in te stellen; op zich zou de standaard distance moeten volstaan maar hier lijkt wel dat er teveel afstand zit tussen kebab-menu tov popover-menu als we het vergelijken met figma design